### ISSUE TITLE
Out of memory issue on createBitmap

### ISSUE THREAD

**Issue by saurabhdhillon** [faruktoptas/FancyShowCaseView#55]
Recenlty i started facing OOM issue on android 6.0.0 on createBitmap, did someone else face the same issue. 
i have used RGB_565 in place of ARGB_888 to reduce the memory usage.

` mCalculator = new Calculator(mActivity, mFocusShape, mView, mFocusCircleRadiusFactor,
                mFitSystemWindows);
        Bitmap bitmap = Bitmap.createBitmap(mCalculator.getBitmapWidth(), mCalculator.getBitmapHeight(),
                Bitmap.Config.RGB_565);
        bitmap.eraseColor(mBackgroundColor);`

**Comment by faruktoptas** [comment]
Why do you need `Calculator`?

**Comment by faruktoptas** [comment]
If you got OOM crash in `createBitmap()` method, this is not about the FancyShowCaseView library. Something else may cause OOM. Can you give more details please?

**Comment by FridayShamblen** [comment]
Having similar issue here. Had some bitmaps in Spannable used in .title option, but removed them and same issue still pops up. Only experiencing this OOM on API 19 and on an emulator. 

Code snippet:
```
   fsv1 = new FancyShowCaseView.Builder(this)
               .focusOn(findViewById(R.id.piece))
               .title("test")
               .build();

    fsv2 = new FancyShowCaseView.Builder(this)
                .focusOn(findViewById(R.id.target))
                .title("Like here ---")
                .build();

    new FancyShowCaseQueue()
                .add(fsv1)
                .add(fsv2)
                .show();
```




Here's the error:
`java.lang.OutOfMemoryError
                                                                            at android.graphics.Bitmap.nativeCreate(Native Method)
                                                                            at android.graphics.Bitmap.createBitmap(Bitmap.java:810)
                                                                            at android.graphics.Bitmap.createBitmap(Bitmap.java:787)
                                                                            at android.graphics.Bitmap.createBitmap(Bitmap.java:754)
                                                                            at me.toptas.fancyshowcase.FancyShowCaseView.show(FancyShowCaseView.java:220)
                                                                            at me.toptas.fancyshowcase.FancyShowCaseQueue.show(FancyShowCaseQueue.java:43)
                                                                            at me.toptas.fancyshowcase.FancyShowCaseQueue.onDismiss(FancyShowCaseQueue.java:65)
                                                                            at me.toptas.fancyshowcase.FancyShowCaseView.removeView(FancyShowCaseView.java:474)
                                                                            at me.toptas.fancyshowcase.FancyShowCaseView$2.onAnimationEnd(FancyShowCaseView.java:340)
                                                                            at android.view.animation.Animation$3.run(Animation.java:374)
                                                                            at android.os.Handler.handleCallback(Handler.java:733)
                                                                            at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                            at android.os.Looper.loop(Looper.java:136)
                                                                            at android.app.ActivityThread.main(ActivityThread.java:5001)
                                                                            at java.lang.reflect.Method.invokeNative(Native Method)
                                                                            at java.lang.reflect.Method.invoke(Method.java:515)
                                                                            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
                                                                            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
                                                                            at dalvik.system.NativeStart.main(Native Method)`

**Comment by faruktoptas** [comment]
@FridayShamblen I will remove the bitmap creation part. It is unnecessary. I think it will fix the OOM.

**Comment by FridayShamblen** [comment]
Thank you sir.

**Comment by faruktoptas** [comment]
@saurabhdhillon @FridayShamblen I fixed it and released v0.1.3
